vissl/data/ssl_dataset.py

```diff
@@ -60,20 +62,24 @@ class GenericSSLDataset(Dataset):
                     }
     """
 
-    def __init__(self, cfg, split, dataset_source_map):
+    def __init__(self, cfg: AttrDict, split: str, dataset_source_map):
         self.split = split
         self.cfg = cfg
         self.data_objs = []
         self.label_objs = []
         self.data_paths = []
         self.label_paths = []
+        self.sub_set = None
```

Comment:
Oh, it will affect both the images and the labels: it effectively reduces the size of the dataset by mapping the range [0, K) to [0, N), where K < N is the DATA_LIMIT value.